bedrock/careers/templates/careers/position.html

```diff
@@ -6,21 +6,21 @@
 
 {% extends "careers/base.html" %}
 
-{% set meta_title = "Mozilla Careers &mdash; " + position.title + " &mdash; Open Positions" %}
-{% set meta_description = "Mozilla Careers &mdash; " + meta_description|truncate(175, end='…') %}
+{% block page_title %}Mozilla Careers — {{ position.title }} — Open Positions{% endblock %}
+{% block page_desc %}meta_description|truncate(175, end='…'){% endblock %}
```

Comment:
This doesn't appear to be working, rendering the following instead

`<meta name="description" content="meta_description|truncate(175, end=&#39;…&#39;)">`
eg view-source:https://www-demo1.allizom.org/en-US/careers/position/gh/3596930/

Reply:
Good catch, will fix